test/sql/test_sink/R/test_select_into_outfile

```diff
@@ -0,0 +1,28 @@
+-- name: test_select_into_outfile
+create table t1 (c1 int, c2 binary);
+-- result:
+-- !result
+insert into t1 select 1, bitmap_to_binary(bitmap_agg(generate_series)) from TABLE(generate_series(1,10));
+-- result:
+-- !result
+select * from t1 into outfile "oss://${oss_bucket}/test_sink/test_files_sink/${uuid0}/" format as parquet;
+-- result:
+-- !result
+create table t2 (c1 int, c2 binary);
```

Comment:
How about using `CREATE TABLE xx AS SELECT xxx` instead?

Reply:
Will modify the case in next pr.